src/EditorFeatures/Test2/FindReferences/FindReferencesTests.OrdinaryMethodSymbols.vb

```diff
@@ -3215,6 +3215,75 @@ End Class
     }
         </Document>
     </Project>
+</Workspace>
+            Await TestAPIAndFeature(input, kind, host)
+        End Function
+
+        <WorkItem(41310, "https://github.com/dotnet/roslyn/issues/41310")>
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function FindReferences_Slice(kind As TestKind, host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+        class C
+        {
+            int Length => 10;
+            int {|Definition:Sli$$ce|}(int start, int length) => start + length;
+            void M()
+            {
+                var c = new C();
+                var o = [|c|][0..^1];
```

Comment:
this seems like an odd thing to show as the result.  i would expect us to match the `[`